index.js

```diff
@@ -28,7 +28,9 @@ function createLoaderPipeline (options, assets) {
                     data: input.code,
                     file: id,
                     outFile: id,
-                    sourceMap: true,
+                    sourceMap: options.hot,
```

Comment:
This should always be set to true, as it will break URL resolving.

Reply:
in production too?

Reply:
Yep! URL resolving needs to be relative to the original source files in order to work. If you for example imported an image in "Icons.scss", without source maps, it will try to import from "main.scss" instead which is incorrect.

**src/main.scss**
```
   @import './impl/icons.scss';
```

**src/impl/icons.scss**
```
.Icon {
     background-image: url('./icon.svg');
}
```

**src/impl/icon.svg**
```
<svg></svg>
```

Reply:
ok